dataclients/kubernetes/kube.go

```diff
@@ -133,6 +133,37 @@ type Options struct {
 	// skipper will load it. The default value for the RouteGroup class is 'skipper'.
 	RouteGroupClass string
 
+	// IngressLabelSelectors is a map of kubernetes labels to their values that must be present on a resource to be loaded
+	// by the client. A label and its value on an Ingress must be match exactly to be loaded by Skipper.
+	// If the value is irrelevant for a given configuration, it can be left empty. The default
+	// value is no labels required.
+	// Examples:
+	//  Config [] will load all Ingresses.
+	// 	Config ["skipper-enabled": ""] will load only Ingresses with a label "skipper-enabled", no matter the value.
+	// 	Config ["skipper-enabled": "true"] will load only Ingresses with a label "skipper-enabled: true"
+	// 	Config ["skipper-enabled": "", "foo": "bar"] will load only Ingresses with both labels while label "foo" must have a value "bar".
+	IngressLabelSelectors map[string]string
```

Comment:
Should this (and others) rather be `IngressLabels` as "selector" is an expression that puts requirements on the label values https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/ ?...

Reply:
In the end, it ends up in a request parameter called `labelSelector` because it's just a specific kind of label selector with the rest not yet implemented.

I don't feel strongly about the naming here. However, if somebody ever decides to implement more from apimachinery to support more of that DSL, `IngressLabels` shorter names will become a tech debt.

Reply:
I agree please don't make it more complicated as needed.

Reply:
> However, if somebody ever decides to implement more from apimachinery to support more of that DSL, IngressLabels shorter names will become a tech debt.

That is what I had in mind: to name proposed configs `FooLabels` and if we ever decide to use DSL - add a new config `FooSelector`.

Lets keep it like that.